### PR TITLE
previewgenerator - allow to clear all previews

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Comment by szaimen** [comment]
This was successfully tested 👍

File: apps/previewgenerator.sh [hunk 1]
```diff
@@ -61,6 +61,17 @@ else
         apt purge ffmpeg -y
     fi
     apt autoremove -y
+    if yesno_box_no "Do you want to remove all previews that were generated until now?
+This will most likely clear a lot of space but your server will need to re-generate the previews \
+if you should opt to reenable previews again."
```
Comment (enoch85) [inline apps/previewgenerator.sh:66]: re-enable?

**Comment by enoch85** [comment]
Great addition!

File: apps/previewgenerator.sh (hunk 1 above)
Comment (szaimen) [reply apps/previewgenerator.sh]: Fixed

**Comment by szaimen** [comment]
> Great addition!

Thanks!

**Pull request merged**